Google.Api.Generator/CodeGenerator.cs

```diff
@@ -192,6 +197,10 @@ internal static class CodeGenerator
                 var unitTestsCsprojContent = CsProjGenerator.GenerateUnitTests(ns);
                 var unitTestsCsprojFilename = $"{unitTestsPathPrefix}{ns}.Tests.csproj";
                 yield return new ResultFile(unitTestsCsprojFilename, unitTestsCsprojContent);
+                // Generate gapic_metadata.json
+                var gapicMetadataJsonContent = MetadataGenerator.GenerateGapicMetadataJson(allServiceDetails);
+                var gapicMetadataJsonFilename = $"{clientPathPrefix}gapic_metadata.json";
```

Comment:
This will generate the metadata in the same directory as the actual clients, which means it would be easy to accidentally ship them.

I think it would be better to generate the metadata file in the root directory - so just use `yield new ResultFile("gapic_metadata.json", gapicMetadataJsonContent);`

Reply:
👍